torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15069,9 +15069,14 @@ def error_inputs_mean(op_info, device, **kwargs):
                    ),),
     OpInfo('lerp',
            dtypes=floating_and_complex_types(),
-           dtypesIfCUDA=floating_and_complex_types_and(torch.half, torch.bfloat16),
-           dtypesIfROCM=floating_and_complex_types_and(torch.half, torch.bfloat16),
+           dtypesIfCUDA=floating_and_complex_types_and(torch.chalf, torch.half, torch.bfloat16),
+           dtypesIfROCM=floating_and_complex_types_and(torch.chalf, torch.half, torch.bfloat16),
```

Comment:
CUDA and ROCm support same dtypes so specifying CUDA should be enough.

Reply:
Ah, okay! I'll remove that statement then. Thanks!